docs/crud/subscriptions.md

```diff
@@ -4,4 +4,59 @@ title: Subscriptions
 sidebar_label: Subscriptions
 ---
 
-## TODO
\ No newline at end of file
+Subscriptions are divided into three groups of changes: **Create**, **Update** and **Delete**. 
+
+- `new<Type>`: subscribe to the [**Create**](./mutations#create) mutation event.
+- `updated<Type>`: subscribe to the [**Update**](./mutations#update) mutation event.
+- `deleted<Type>`: subscribe to the [**Delete**](./mutations#delete) mutation event.
+
+Graphback generates a subscription handler and filter input for each CRUD mutation field to let you subscribe to mutation events and receive real-time updates.
+
+```graphql
+""" @model """
+type Note {
+  id: ID!
+  title: String!
+  likes: Int
+}
+
+// highlight-start
+type Subscription {
+  newNote: Note!
+  updatedNote: Note!
+  deletedNote: Note!
+}
+// highlight-end
+```
+
+### Examples
+
+Subscribing to a [`create`](./mutations#create) event on `Note`:
+
+```graphql
+newNote {
+  id
+  title
+  likes
+}
+```
```

Comment:
```graphql
subscription {
newNote {
  id
  title
  likes
}
}
```
?

Likewise for update and delete

Reply:
Done now, for queries and mutations also.